### PR TITLE
clean up pom and fix warnings

### PR DESCRIPTION
**Pull request by sshaaf**
removing warnings in the pom.xml for backend and parent.

### CODE REVIEW

File: backend/pom.xml [hunk 1]
```diff
@@ -4,37 +4,21 @@
...
     <name>frdemo-backend</name>
     <packaging>war</packaging>
-    <parent>
-        <groupId>org.jboss</groupId>
-        <artifactId>jboss-parent</artifactId>
-        <version>37</version>
-    </parent>
     <properties>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
```
Comment (kittylyst) [inline backend/pom.xml:15]: Should we align on version 17?

**Review by kittylyst — APPROVED** [review]
Up to you what to do about version number for the main project. Other than that, LGTM.

File: backend/pom.xml (hunk 1 above)
Comment (jasondlee) [reply backend/pom.xml:15]: I like 17.

Comment (sshaaf) [reply backend/pom.xml:15]: Getting the following when I move it to 17.. hence I moved simulator also back to 11 for now.
``` 
19:14:43,497 ERROR [org.jboss.msc.service.fail] (MSC service thread 1-3) MSC000001: Failed to start service org.wildfly.clustering.infinispan.cache-container-configuration.hibernate: org.jboss.msc.service.StartException in service org.wildfly.clustering.infinispan.cache-container-configuration.hibernate: java.lang.ExceptionInInitializerError
        at org.wildfly.clustering.service@26.1.1.Final//org.wildfly.clustering.service.FunctionalService.start(FunctionalService.java:66)
        at org.jboss.msc@1.4.13.Final//org.jboss.msc.service.ServiceControllerImpl$StartTask.startService(ServiceControllerImpl.java:1739)
        at org.jboss.msc@1.4.13.Final//org.jboss.msc.service.ServiceControllerImpl$StartTask.execute(ServiceControllerImpl.java:1701)
        at org.jboss.msc@1.4.13.Final//org.jboss.msc.service.ServiceControllerImpl$ControllerTask.run(ServiceControllerImpl.java:1559)
        at org.jboss.threads@2.4.0.Final//org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
        at org.jboss.threads@2.4.0.Final//org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:1990)
        at org.jboss.threads@2.4.0.Final//org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1486)
        at org.jboss.threads@2.4.0.Final//org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1363)
        at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.ExceptionInInitializerError
        at org.wildfly.clustering.marshalling.spi@26.1.1.Final//org.wildfly.clustering.marshalling.spi.util.UtilExternalizerProvider.<clinit>(UtilExternalizerProvider.java:78)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at java.base/java.lang.Class.getEnumConstantsShared(Class.java:3837)
        at java.base/java.lang.System$2.getEnumConstantsShared(System.java:2284)
        at java.base/java.util.EnumSet.getUniverse(EnumSet.java:408)
        at java.base/java.util.EnumSet.noneOf(EnumSet.java:111)
        at java.base/java.util.EnumSet.allOf(EnumSet.java:132)
        at org.wildfly.clustering.marshalling.jboss@26.1.1.Final//org.wildfly.clustering.marshalling.jboss.DefaultExternalizerProviders.<init>(DefaultExternalizerProviders.java:54)
        at org.wildfly.clustering.marshalling.jboss@26.1.1.Final//org.wildfly.clustering.marshalling.jboss.DefaultExternalizerProviders.<clinit>(DefaultExternalizerProviders.java:46)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at java.base/java.lang.Class.getEnumConstantsShared(Class.java:3837)
        at java.base/java.lang.System$2.getEnumConstantsShared(System.java:2284)
```

Comment (kittylyst) [reply backend/pom.xml:15]: That might be a post-16 visibility problem. Either way, that shouldn't delay the merge. Can we add an issue to backlog please, and I'll merge this now.

**Pull request merged**